examples/data-objects/table-document/package.json

```diff
@@ -65,9 +65,10 @@
     "@fluidframework/local-driver": "^0.37.0",
     "@fluidframework/mocha-test-setup": "^0.37.0",
     "@fluidframework/runtime-utils": "^0.37.0",
-    "@fluidframework/server-local-server": "^0.1021.0-0",
+    "@fluidframework/test-drivers": "^0.37.0",
```

Comment:
>fluidframework [](start = 6, length = 14)

i don't like that we need this dep. hoping with your dynamic loading we can get rid of it :)